app/src/main/java/org/sopt/dosopttemplate/data/local/dao/UserInfoDao.kt

```diff
@@ -0,0 +1,17 @@
+package org.sopt.dosopttemplate.data.local.dao
+
+import androidx.room.Dao
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.Query
+import org.sopt.dosopttemplate.data.local.entity.UserInfoEntity
+
+@Entity
+@Dao
+interface UserInfoDao {
+    @Insert
+    fun saveUserInfo(userInfoEntity: UserInfoEntity)
+
+    @Query("SELECT * FROM user_info LIMIT 1")
+    fun getUserInfo(): UserInfoEntity
```

Comment:
```suggestion
    fun getUserInfo(): Flow<UserInfoEntity>
```
room db는 flow를 사용 할 수 있는 걸로 알고 있어요!